packages/app/src/__deprecated__/renderer/locales/default/en-US.json

```diff
@@ -652,9 +652,15 @@
   "module.overview.systemUpdateUpToDate": "You’re up to date.",
   "module.overview.systemVersion": "MuditaOS version",
   "module.overview.systemVersionTitle": "Current version:",
-  "module.overview.updateAvailableButton": "Download now",
-  "module.overview.updateAvailableDescription": "MuditaOS v{version} (Updated on {date})",
+  "module.overview.updateAvailableButton": "Download all",
+  "module.overview.updateAvailableDescription": "You can download all the MuditaOS versions available to you with one click.",
   "module.overview.updateAvailableMessage": "New update is available!",
+  "module.overview.updateAvailableAboutUpdatesTitle": "ABOUT UPDATES",
+  "module.overview.updateAvailableAboutUpdatesOsVersionDescription": "MuditaOS v{version} ({date})",
+  "module.overview.updateAvailableAboutUpdatesTargetOsVersionSubDescription": "the device will be upgraded to this version",
+  "module.overview.updateAvailableCautionSectionTitle": "Caution",
+  "module.overview.updateAvailableCautionSectionSequenceUpdateDescription": "1. The device cannot be used or disconnected during sequential update.\n2. You will need to enter your phone passcode during the update.\n3. The phone may restart several times during the update process.\n4. The update process may take longer than usual.",
+  "module.overview.updateAvailableCautionSectionSingleUpdateDescription": "1. The device cannot be used or disconnected during update.\n2. The phone may restart several times during the update process.",
```

Comment:
Can't we shorten the keys? It is difficult to read them.

Reply:
after making keys shortened I'm not sure that will be known what a section/component key is. Maybe decouple the overview to the component instead of just only view? 🤔 

for example 
```
"module.overviewUpdateAvailableCautionSection.Title": "Caution",
```

Reply:
How about something like `module.overview.updateAvailableSingleUpdateDescription`? A little bit shorter, (at least for me) it still suggests where it is used, follows the pattern with `module.component.xxx`.

Reply:
https://github.com/mudita/mudita-center/pull/1049/commits/b1e89e36ec965f086ebdbf036cf35ef7965c500d